daemon/execdriver/native/driver.go

```diff
@@ -51,6 +52,15 @@ func NewDriver(root, initPath string, options []string) (*driver, error) {
 		return nil, err
 	}
 
+	if apparmor.IsEnabled() {
+		apparmor_policies := []string{"docker-default", "docker-unconfined"}
+		for _, policy := range apparmor_policies {
+			if err := hasAppArmorProfileLoaded(policy); err != nil {
+				return nil, fmt.Errorf("AppArmor enabled on system but the %s profile is not loaded. Install policies from contrib/apparmor (https://goo.gl/CjGtgC) and try again.", policy)
```

Comment:
The problem is that if we don't error here it *does* break people. This is why the docker-unconfined patch was reverted. The policy didn't exist and the daemon just moved on, then broken. Similarly, if docker-default isn't there, the daemon would similarly break.

The alternative for 'docker-unconfined' is that if it doesn't exist, that (for one release) we could fallback to 'unconfined'. Although that might cause more harm, especially if we're okay with the installation / usage docs (and packages).